lib/act.js

```diff
@@ -9,8 +9,9 @@ var call;
 var config;
 var widget;
 var cache;
+var debug = require('debug')('fh-mbaas-api:act');
```

Comment:
@wojta @david-martin  I would suggest that a wrapper for debug might be better than using the pure module? Logs with timestamps would be much more valuable.

```js
// debug-wrapper.js
var debug = require('debug')

module.exports = function getLogger(name) {
  var log = debug(name);

  return function doLog () {
    var args = Array.prototype.slice.call(arguments);

    args.unshift(new Date().toJSON()); // add a date stamp to each log

    log.apply(log, args);
  }
}
```

This print something like this:

```
testing 2017-03-24T15:16:51.370Z hello world
```